src/main/java/core/basesyntax/service/operation/ReturnOperationHandler.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void apply(Map<String, Integer> storage, FruitTransaction transaction) {
+        storage.merge(
+                transaction.getFruit(),
+                transaction.getQuantity(),
+                Integer::sum
+        );
+    }
```

Comment:
The class `ReturnOperationHandler` should be placed in the `strategy` package according to the checklist. Please move this class to the correct package to maintain a clean project structure.